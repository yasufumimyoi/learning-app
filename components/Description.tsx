const Description = () => (
  <div>
    <h3 className="text-center text-lg md:text-2xl mb-10">
      現在50個以上のMissionが用意されています。
    </h3>
    <div className="flex justify-center">
      <div className="text-sm">
        <p className="mb-4">
          YouTube動画なので完全無料。無料とは思えないほどコンテンツの質が高いのでオススメ
        </p>
        <p className="mb-4">
          テキストよりもイメージしやすい。実際のコーディングも見れるので、気付きがあるかも？
        </p>
        <p className="mb-4">
          学習時間が取れないなら通勤・通学のスキマ時間を活用して学習を進めてみませんか？
        </p>
      </div>
    </div>
  </div>
)

export default Description
