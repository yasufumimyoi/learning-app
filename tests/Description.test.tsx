import { render } from '@testing-library/react'
import Description from '../components/Description'

it('Descriptionのsnapshot"', () => {
  const component = render(<Description />)
  expect(component).toMatchSnapshot()
})
