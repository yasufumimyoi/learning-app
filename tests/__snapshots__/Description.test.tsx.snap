// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Descriptionのsnapshot" 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <h3
          class="text-center text-lg md:text-2xl mb-10"
        >
          現在50個以上のMissionが用意されています。
        </h3>
        <div
          class="flex justify-center"
        >
          <div
            class="text-sm"
          >
            <p
              class="mb-4"
            >
              YouTube動画なので完全無料。無料とは思えないほどコンテンツの質が高いのでオススメ
            </p>
            <p
              class="mb-4"
            >
              テキストよりもイメージしやすい。実際のコーディングも見れるので、気付きがあるかも？
            </p>
            <p
              class="mb-4"
            >
              学習時間が取れないなら通勤・通学のスキマ時間を活用して学習を進めてみませんか？
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <h3
        class="text-center text-lg md:text-2xl mb-10"
      >
        現在50個以上のMissionが用意されています。
      </h3>
      <div
        class="flex justify-center"
      >
        <div
          class="text-sm"
        >
          <p
            class="mb-4"
          >
            YouTube動画なので完全無料。無料とは思えないほどコンテンツの質が高いのでオススメ
          </p>
          <p
            class="mb-4"
          >
            テキストよりもイメージしやすい。実際のコーディングも見れるので、気付きがあるかも？
          </p>
          <p
            class="mb-4"
          >
            学習時間が取れないなら通勤・通学のスキマ時間を活用して学習を進めてみませんか？
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
